src/main/java/no/nav/familie/integrasjoner/dokarkiv/metadata/BarnetrygdVedtakVedleggMetadata.kt

```diff
@@ -0,0 +1,19 @@
+package no.nav.familie.integrasjoner.dokarkiv.metadata
+
+import no.nav.familie.integrasjoner.dokarkiv.client.domene.JournalpostType
+import org.springframework.stereotype.Component
+
+@Component
+object BarnetrygdVedleggMetadata : DokumentMetadata {
+
+    override val journalpostType: JournalpostType = JournalpostType.UTGAAENDE
+    override val fagsakSystem: String? = "BA"
+    override val tema: String = "BAR"
+    override val behandlingstema: String? = null
```

Comment:
Det er nå laget en enum for behandlingstema for barnetrygdkoder, men det er ikke sikkert det har noe for seg å bruke den her.  Mer sånn til info :-) 
https://github.com/navikt/familie-kontrakter/pull/83/files

Reply:
Fint å bruke enumen tenker jeg!

Reply:
Det er jo supert :D Vi skal nok ikke ha behandlingstema på vedlegg, men hoveddokumentet det legges ved ser jeg har et "Barnetrygd"-tema som ikke ligger inne som enum, så legger inn en [pr i kontrakter](https://github.com/navikt/familie-kontrakter/pull/84) på det og oppdaterer når det er klart 👍